Objective-J/Runtime.js

```diff
@@ -1020,9 +1020,39 @@ GLOBAL(method_getName) = function(/*Method*/ aMethod)
     return aMethod.method_name;
 }
 
-GLOBAL(method_getTypes) = function(/*Method*/ aMethod)
+GLOBAL(method_copyReturnType) = function(/*Method*/ aMethod)
 {
-    return aMethod.method_types;
+    var types = aMethod.method_types;
+
+    if (types)
+    {
+        var argType = types[0];
+
+        return argType != NULL ? argType : NULL;
+    }
+    else
+        return NULL;
+}
+
+GLOBAL(method_copyArgumentType) = function(/*Method*/ aMethod, /*unsigned int*/ index)
```

Comment:
Just wondering, do we should to follow cocoa here ? Cocoa the first index and second index are the _cmd and self.